ddtrace/context.py

```diff
@@ -52,6 +53,11 @@ def span_id(self):
             return self._parent_span_id
 
     @property
+    def service(self):
```

Comment:
would be better to make it "private" too

Reply:
privatized

Reply:
removed so marking resolved